tools/add_triage_signature.py

```diff
@@ -569,10 +569,14 @@ def isVMware(host):
         status_info = cluster['status_info']
         openshift_version = cluster['openshift_version']
         report = ""
-        if "waiting for console" in status_info and "4.7" in openshift_version:
-            if any(isVMware(host) for host in cluster['hosts']):
-                report = "h2. Waiting for console timeout probably due to VMware hosts on OCP 4.7\n"
-                report += "See: https://bugzilla.redhat.com/1926345\n"
+        if ("waiting for console" in status_info and
+                "4.7" in openshift_version and
+                any(isVMware(host) for host in cluster['hosts'])):
+            report = "\n".join([
+                "h2. Waiting for console timeout probably due to VMware hosts on OCP 4.7",
+                "You can mark it as caused by issue [MGMT-4454|https://issues.redhat.com/browse/MGMT-4454]",
```

Comment:
Can we set the issue link automatically as well? :)

Reply:
Probably we would do that next
I think we'll discuss it in the next triage weekly. Each signature have its own level of strength, such that some should just be informing, some can be connected automatically to tickets, some can just close the ticket, etc.